x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/332_connector_update_filtering.yml

```diff
@@ -390,74 +193,7 @@ setup:
       connector.update_filtering:
         connector_id: test-connector
         body:
-          filtering:
-            - active:
-                advanced_snippet:
-                  created_at: "this-is-not-a-datetime-!!!!"
-                  updated_at: "2023-05-25T12:30:00.000Z"
-                  value: {}
-                rules:
-                  - created_at: "2023-05-25T12:30:00.000Z"
-                    field: _
-                    id: RULE-ACTIVE-0
-                    order: 0
-                    policy: include
-                    rule: regex
-                    updated_at: "2023-05-25T12:30:00.000Z"
-                    value: ".*"
-                validation:
-                  errors: []
-                  state: valid
-              domain: DEFAULT
-              draft:
-                advanced_snippet:
-                  created_at: "2023-05-25T12:30:00.000Z"
-                  updated_at: "2023-05-25T12:30:00.000Z"
-                  value: {}
-                rules:
-                  - created_at: "2023-05-25T12:30:00.000Z"
-                    field: _
-                    id: RULE-DRAFT-0
-                    order: 0
-                    policy: include
-                    rule: regex
-                    updated_at: "2023-05-25T12:30:00.000Z"
-                    value: ".*"
-                validation:
-                  errors: []
-                  state: valid
-            - active:
-                advanced_snippet:
-                  created_at: "2021-05-25T12:30:00.000Z"
-                  updated_at: "2021-05-25T12:30:00.000Z"
-                  value: {}
-                rules:
-                  - created_at: "2021-05-25T12:30:00.000Z"
-                    field: _
-                    id: RULE-ACTIVE-1
-                    order: 0
-                    policy: include
-                    rule: regex
-                    updated_at: "2021-05-25T12:30:00.000Z"
-                    value: ".*"
-                validation:
-                  errors: []
-                  state: valid
-              domain: TEST
-              draft:
-                advanced_snippet:
-                  created_at: "2021-05-25T12:30:00.000Z"
-                  updated_at: "2021-05-25T12:30:00.000Z"
-                  value: {}
-                rules:
-                  - created_at: "2021-05-25T12:30:00.000Z"
-                    field: _
-                    id: RULE-DRAFT-1
-                    order: 0
-                    policy: exclude
-                    rule: regex
-                    updated_at: "2021-05-25T12:30:00.000Z"
-                    value: ".*"
-                validation:
-                  errors: []
-                  state: valid
+          rules: [ ]
```

Comment:
Do we've a test to verify that the `domain` cannot be updated?

Reply:
Changed the logic, user can't edit domain via API now